Gandalf/build.sh

```diff
@@ -1,119 +1,98 @@
 #!/bin/bash
+# 
+# Packages "Gandalf" into a deb
+#
+# Follow https://google.github.io/styleguide/shell.xml
+# for the most part (You can ignore some, like error checking for mv)
 
-#Please run with a Linux flavor. macOS makes a lot of dummy files.
-
-# Aliases
-
-basename="io.github.ethanrdoesmc.gandalf102" #identifier (ios internal name)
-conflictfile="conflicts102.txt" # Name of conflictsfile
-scriptname="gandalf102" #name of hold script
-productname="Gandalf 102" #name of gandalf for user (shown in terminal)
-
-# Define controlfile (set variables)
-packagename="Gandalf for Yalu102" # Name shown in Cydia
+# Config
+PKG_VERSION="2.4" #Bump this everytime you update something.
+CONFLICTS_FILE="conflicts102.txt"
 
-########################
-packageversion="2.4" # Here edit the versionnumber
-########################
+#DO NOT TOUCH! (Unless you have a good reason...)
+#Variable format is "PKG_FIELDNAME"
+PKG_PACKAGE="io.github.ethanrdoesmc.gandalf102"
+PKG_NAME="Gandalf for Yalu102"
+PKG_DESCRIPTION="Some tweaks may break jailbreaks. Let this tweak say
+  \"You Shall Not Pass!\" to incompatible tweaks and you can sit back and have
+  fun with your jailbreak."
+PKG_DEPICTION="https://ethanrdoesmc.github.io/gandalf/depictions/?p=io.github.ethanrdoesmc.gandalf102"
+PKG_MAINTAINER="EthanRDoesMC <ethanrdoesmc@gmail.com>"
+PKG_AUTHOR="EthanRDoesMC <ethanrdoesmc@gmail.com>"
+PKG_SECTION="Manager_addons"
+PKG_DEPENDS="firmware (>=10.0), sudo, com.officialscheduler.mterminal"
+PKG_REPLACES="com.enduniverse.cydiaextenderplus, com.github.ethanrdoesmc.gandalf,
+  com.github.ethanrdoesmc.gandalf102"
+PKG_ARCHITECTURE='iphoneos-arm'
+PKG_BREAKS=$(cat ${CONFLICTS_FILIE} | sed ':a;N;$!ba;s/\n/,\ /g')
 
-packagereplaces="com.enduniverse.cydiaextenderplus, com.github.ethanrdoesmc.gandalf, com.github.ethanrdoesmc.gandalf102"
-packagedescription='Some tweaks may break jailbreaks. Let this tweak say "You Shall Not Pass!" to incompatible tweaks and you can sit back and have fun with your jailbreak.'
-packagedepiction='https://ethanrdoesmc.github.io/gandalf/docs/depictions/?p=io.github.ethanrdoesmc.gandalf102'
-maintainer="EthanRDoesMC <ethanrdoesmc@gmail.com>"
-author="EthanRDoesMC <ethanrdoesmc@gmail.com>"
-depends="firmware (>=10.0), sudo, com.officialscheduler.mterminal"
-section="Manager_Addons"
+#Script specific variables.
+GANDALF_COMMAND_NAME="gandalf102"
 
-# Show user what he's trying to compile
+#Main script
 
+#Start message
+echo "Started packaging ${PKG_NAME}"
```

Comment:
Yes. But I do think that there will be also some people who don't know what this script does since they only added e.g. A conflict to conflicts.txt let's say for gandalf_portal. But now they run build.sh without changing anything, and the script creates a controlfile for Gandalf102. So they will most likely compile it the wrong way.